README.md

```diff
@@ -22,18 +20,20 @@ This application will be responsible for creating a User Interface that will all
 ### Requirements:
 
 - 1 React / JavaScript / Redux application.
-  - 1 Page for viewing the edited version of the Ruleset.
-  - OPTIONAL 1 Page for viewing the read-only version of the Ruleset.
+  - 1 Page for creating a new Ruleset.
+  - 1 Page for viewing the edited version of a Ruleset.
```

Comment:
Update?
"1 Page for editing Rulesets that have been created."
Yes, the page is for viewing the edited Ruleset but it's mainly for editing the ruleset not just viewing it. 
The wording makes it seem like a page to see an edited version, not a page to edit.